src/token.ts

```diff
@@ -95,3 +91,65 @@ const refreshAccessToken = (client_id: string, client_secret: string, refresh_to
         })
     })
 }
+
+export const getAccessTokenByServiceAccount = async (service_account: ServiceAccount, sub: string) => {
```

Comment:
@ice1829 qui sarebbe utile un commento che indichi che il sub e' l'utente per il quale il service account richiedera' accesso in delega